client/src/components/Login/NewUserConfirmation.vue

```diff
@@ -18,9 +18,9 @@
                                 connect this account via <strong>User Preferences</strong>.
                                 <a
                                     href="https://galaxyproject.org/authnz/use/oidc/idps/custos/#link-an-existing-galaxy-account"
-                                    target="_blank"
-                                    >More details here.</a
-                                >
+                                    target="_blank">
+                                    More details here.
```

Comment:
Doesn't matter here in the end, but be careful with changes to lines like this -- the spaces can be significant and impact the display.